test/fudge/log_test.clj

```diff
@@ -126,33 +134,33 @@
       (let [logger (plain-logger)
             result (with-out-str
                      (log logger :info "foo"))]
-        (is (= "2017-05-12T18:01 [fudge.log-test:128] [info] foo\n" result))))
+        (is (= "2017-05-12T18:01 [fudge.log-test:136] [info] foo\n" result))))
```

Comment:
This kind of tweak of line numbers suggests to me that the macro stuff should be a sugar layer (possibly in `fudge.sugar` ;-) atop the functions, and we don't use it for the majority of the testing.